iroh-rpc-client/src/status.rs

```diff
@@ -196,12 +199,32 @@ impl StatusTable {
     }
 }
 
+impl Iterator for StatusTable {
```

Comment:
This is a bit strange. The usual pattern is to create a new type that holds a reference to `StatusTable` as the iterator

Reply:
in2it, will change